domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ResponseRejectionTest.java

```diff
@@ -29,6 +29,26 @@ public void shouldBeValidWhenReasonNotPresent() {
         assertThat(response).hasSize(0);
     }
 
+    @Test
+    public void shouldBeValidWithFreeMediationResponse() {
+        ClaimantResponse claimantResponse = SampleClaimantResponse.validRejectionWithFreeMediation();
+
+        Set<String> response = validate(claimantResponse);
+
+        assertThat(response).hasSize(0);
+    }
+
+    @Test
+    public void shouldBeInvalidWhenMediationPhoneNumberIsTooLong() {
+        ClaimantResponse claimantResponse = ResponseRejection
+            .builder()
+            .mediationPhoneNumber("aaaabbbbasdfafasfdasdfagasdfasfgasfdsasdghsf").build();
```

Comment:
It would be more cleaner to use: `RandomStringUtils.randomAlphabetic(31)`

Reply:
much better :), i will do that